packages/mdc-chips/chip-set/index.ts

```diff
@@ -23,4 +23,8 @@
 
 export * from './adapter';
 export * from './component';
+export {
```

Comment:
Another idea would be for all of them do something like: `export * as chipSetConstants from './constants';`. That way you don't have to rename all the individual symbols.

Reply:
ah I like that idea...I"ll try that out :)

Reply:
I don't think that syntax actually works.

Reply:
Oh interesting looks like you're right. I assumed it would since that style syntax works for import. In that case I think you could accomplish the same thing like this:

```ts
import * as chipSetConstants from './constants';
export {chipSetConstants};
```

Reply:
Ah - I did try that but I forgot the `*`...Not as clean as your original, but better than manually naming everything.